lib/alertmanager_api/silence_details.py

```diff
@@ -196,16 +196,16 @@ def add_from_json(self, json_doc_l):
             # Set the comment
             comment = json_doc["comment"]
 
-            # Extract the instance name from the matchers list (look for a 'name' == 'instance')
+            # Extract the hostname name from the matchers list (look for a 'name' == 'hostname')
             for matcher in json_doc.get("matchers", []):
-                if matcher["name"] == "instance":
-                    instance_name = matcher["value"]
+                if matcher["name"] == "hostname":
+                    hostname = matcher["value"]
                     break
```

Comment:
actually I'm not sure this works. 
As @lizsalmon  - We have 2 alerts - one which stores hv name under `matchers.name = "instance"`` and one that has `matchers.name = "hostname"` - we need to grab both silences - so you need to keep the previous one and add elif